app/controllers/actions/DataRequiredAction.scala

```diff
@@ -31,8 +32,9 @@ class DataRequiredAction(lrn: LocalReferenceNumber, config: FrontendAppConfig)(i
 
   override protected def refine[A](request: OptionalDataRequest[A]): Future[Either[Result, DataRequest[A]]] =
     request.userAnswers match {
-      case Some(userAnswers) if userAnswers.status != SubmissionState.Submitted =>
+      case Answers(userAnswers) if userAnswers.status != SubmissionState.Submitted =>
         Future.successful(Right(DataRequest(request.request, request.eoriNumber, userAnswers)))
+      case UserAnswersResponse.NotAcceptable => Future.successful(Left(Redirect(controllers.routes.DraftNoLongerAvailableController.onPageLoad(lrn))))
```

Comment:
Remove DraftNoLongerAvailableController and change this to
`config. draftNotAvailableUrl`